### PR TITLE
Dev 147 create partners page in nextjs

### PR DESCRIPTION
**Pull request by simon-hng**
First implementation of partner page in nextjs.

### CODE REVIEW

File: pages/partners.js [hunk 1]
```diff
@@ -1,27 +1,135 @@
...
-			{/*<PartnersRow imageSrc={'/assets/partners_sponsors/applied-ai.svg'} companyName={'appliedAI'} />*/}
-			{/*<PartnersRow imageSrc={'/assets/partners_sponsors/UnternehmerTUM.webp'} companyName={'UnternehmerTUM'} />*/}
-
-		</>
-	);
-}
\ No newline at end of file
+  const cards = [
+    {
+      img: "https://www.tum-ai.com/assets/img/mainpage_slideshow/alelphalpha_workshop_group.jpg",
```
Comment (simon-hng) [inline pages/partners.js]: Images need to be downloaded and added in assets but I was too lazy to do it yet

File: pages/partners.js [hunk 2]
```diff
@@ -1,27 +1,135 @@
...
+    <>
+      <Header1
+        imageSrc={"/assets/partners.jpg"}
+        title="Become a partner of TUM.ai"
+        subtitle="Get access to Munich's largest student talent pool of AI enthusiasts!"
+      />
+
+      <section className="flex flex-col items-center py-8 container mx-auto">
+        <h2 className="text-uppercase text-text text-4xl font-bold mb-8">
+          Partners <span className="text-[#5a2de5]">we have collaborated</span>{" "}
```
Comment (simon-hng) [inline pages/partners.js]: We define various shades of purple in `tailwind.config` and yet non of them match the one on the website.

Color usage needs to be documented somewhere

File: pages/partners.js [hunk 3]
```diff
@@ -1,27 +1,135 @@
...
+      </section>
+
+      <section className="bg-[url('https://www.tum-ai.com/assets/img/mainpage_slideshow/alephalpha_workshop.jpg')]">
+        <div className="py-8 md:py-16 grid md:grid-cols-2 gap-4 container mx-auto">
+          {cards.map((card) => (
+            <article
+              key={card.title}
+              className="bg-[#223358] text-white text-center h-full p-4"
+            >
+              <img src={card.img} />
```
Comment (MunzerDw) [inline pages/partners.js]: Please use nextjs Image component :).

File: pages/partners.js (hunk 2 above)
Comment (MunzerDw) [reply pages/partners.js]: Sure, how about we call the colors "primary", "secondary", "tertiary" etc.? You can already implement that in your branch.

File: pages/partners.js (hunk 1 above)
Comment (MunzerDw) [reply pages/partners.js]: hahah no worries

**Comment by MunzerDw** [comment]
I know you are not done yet, but it looks great so far! Thanks for your work.
I will adapt my branch to yours once we merge this branch.

**Review by MunzerDw — CHANGES_REQUESTED** [review]
looks great, thanks!
Could you please move the sections into different components?

File: pages/partners.js (hunk 3 above)
Comment (simon-hng) [reply pages/partners.js]: outdated - already done

**Comment by simon-hng** [comment]
I don't think creating components out of the sections here is the right step and we should merge this ASAP. We can still create components out of the sections here in the future when the need arises

**Review by MunzerDw — APPROVED** [review]

**Pull request merged**